app/Console/Commands/AllocateStandForArrival.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Vatsim\NetworkAircraft;
+use App\Services\StandService;
+use Illuminate\Console\Command;
+
+class AllocateStandForArrival extends Command
+{
+    public const ALLOCATE_STANDS_CONFIG_KEY = 'stands.auto_allocate';
```

Comment:
This feels like an unnecessary abstraction

Reply:
Removed abstraction